doc/lpvs-api.yaml

```
@@ -48,6 +48,55 @@ paths:
               schema:
                 $ref: '#/components/schemas/WebhookResponseForbidden'
 
+  /scan/{gitHubDomain}/{gitHubOrg}/{gitHubRepo}/{prNumber}:
```

I don't think that `gitHubDomain` is necessary. It is already set to 'github.api.url' and if detailed regulations are not provided, errors are likely to happen.

```suggestion
  /scan/{gitHubOrg}/{gitHubRepo}/{prNumber}:
```

---

I agree with you. The code is updated.